BB-CEST/_Not_Clustered/BB_Car_Simulator.md

```diff
@@ -0,0 +1,84 @@
+
+# Car Simulator
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-CEST
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+None
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+Cross-Layer
+
+## Known Implementation
+
```

Comment:
There will be an Eclipse SDV project soon which at least partially can implement this BB (formerly called kanto.auto, but will publish with a different name still tbd).
Also the Eclipse OpenDUT project might play a role for this building block

Reply:
Agreed. Will be taken care of in the next pull request.

Reply:
The _Not_Clustered folder holds BB that have no FC or Layer specified. This folder has to be removed in the future, when it is clear where to move "BB_Car_Simulator.md".

Reply:
The "_Not_Clustered" folder holds BB that don't yet have a specified FC. This folder has to be removed as soon as we decide on an appropiate Layer and FC for "BB_Car_Simulator.md"